metaserver/src/test/scala/tests/search/SymbolIndexTest.scala

```diff
@@ -51,28 +52,70 @@ object SymbolIndexTest extends MegaSuite {
         line: Int,
         column: Int,
         expected: String
-    ): Unit = {
-      val term = indexer.findSymbol(path, line, column)
+    ): Option[Symbol] = {
+      val symbol = indexer.findSymbol(path, line, column)
+      Predef.assert(
+        symbol.isDefined,
+        s"Symbol not found at $path:$line:$column."
```

Comment:
I think we can add "Did you run scalametaEnableCompletions from sbt?" here too, just in case.

Reply:
Fixed in 3548ab933e71621b13af599261d5da2f16e65567